### DISCUSSION TITLE
Daily challenge intro screen should only be shown once per day/play session

### DISCUSSION THREAD

**Discussion by IceBlossom** [ppy/osu#29443] [Ideas] [answered]
For people who pop in and out of DC chat multiple times a day, I feel like reducing the intro screen to 1 instance per day/session would be nice. Probably not "a whole new settings option" worthy, but perhaps some nice QoL.

**Comment by peppy** [comment]
Probably.

**Comment by Gabixel** [comment, marked as answer]
#29542
